signature/policy_config_test.go

```diff
@@ -84,17 +85,37 @@ func TestDefaultPolicy(t *testing.T) {
 }
 
 func TestDefaultPolicyPath(t *testing.T) {
-	// The common case
-	path := defaultPolicyPath(nil)
-	assert.Equal(t, systemDefaultPolicyPath, path)
-	// There is a context, but it does not override the path.
-	path = defaultPolicyPath(&types.SystemContext{})
-	assert.Equal(t, systemDefaultPolicyPath, path)
-
-	// Path overridden
+
 	const nondefaultPath = "/this/is/not/the/default/path.json"
-	path = defaultPolicyPath(&types.SystemContext{SignaturePolicyPath: nondefaultPath})
-	assert.Equal(t, nondefaultPath, path)
+	const rootPrefix = "/root/prefix"
+
+	for _, c := range []struct {
+		ctx      *types.SystemContext
+		expected string
+	}{
```

Comment:
Test what? `defaultPolicyPath` does not use the home directory.  (re: the other conversation about environment variables, that happens for `ATOMIC_CONF` in `docker/lookaside.go:atomicConfPath`, but not in this PR).

Do you mean testing that an explicit `SignaturePolicyPath` is not expanded if it contains `$HOME`?